var/spack/repos/builtin/packages/py-ply/package.py

```diff
@@ -28,6 +28,7 @@
 class PyPly(PythonPackage):
     """PLY is nothing more than a straightforward lex/yacc implementation."""
     homepage = "http://www.dabeaz.com/ply"
-    url      = "http://www.dabeaz.com/ply/ply-3.8.tar.gz"
+    url      = "https://github.com/dabeaz/ply/archive/3.11.tar.gz"
 
-    version('3.8', '94726411496c52c87c2b9429b12d5c50')
+    version('3.11', '6465f602e656455affcd7c5734c638f8')
+    version('3.8', '94726411496c52c87c2b9429b12d5c50', url='http://www.dabeaz.com/ply/ply-3.8.tar.gz')
```

Comment:
Can you run `spack info py-ply` and make sure this actually works? I'm wondering if it might suffer from #2737.

Reply:
And indeed it is.

Reply:
Should run fine as soon as #8565 gets merged.